openfisca_core/taxbenefitsystems.py

```diff
@@ -63,13 +70,17 @@ class TaxBenefitSystem(object):
     cache_blacklist = None
     decomposition_file_path = None
 
-    def __init__(self, entities, legislation_json = None):
+    def __init__(self, entities, legislation = None):
+        """
+        :param entities: Entities used by the tax benefit system.
+        :param legislation: Parameters defined for the tax benefit system. Can be defined later.
+        """
         # TODO: Currently: Don't use a weakref, because they are cleared by Paste (at least) at each call.
-        self.compact_legislation_by_instant_cache = {}  # weakref.WeakValueDictionary()
+        self.legislation_at_instant_cache = {}  # weakref.WeakValueDictionary()
```

Comment:
As we are renaming it, should we be opportunistic and rename it to `parameters_at_instant_cache` ?